src/front/app/components/challenges/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Please rename this component file name regarding to 
https://github.com/airbnb/javascript/tree/master/react#naming